Makefile

```diff
@@ -42,14 +42,14 @@ optimize:
 	$(CONTAINER_RUNTIME) run --rm -v $(CURDIR):/code:Z \
 		--mount type=volume,source=ats-smart-contract_cache,target=/code/target \
 		--mount type=volume,source=ats-smart-contract_registry_cache,target=/usr/local/cargo/registry \
-		cosmwasm/rust-optimizer:0.12.12
+		cosmwasm/rust-optimizer:0.15.0
```

Comment:
I think the `rust-optimizer` is technically deprecated and `cosmwasm/optimizer` should be used instead https://github.com/CosmWasm/optimizer?tab=readme-ov-file#cosmwasm-optimizing-compiler

Reply:
Oh ok, wasn't aware of that, will update 👍